src/boot_O2_g3/fault.c

```diff
@@ -112,27 +165,39 @@ void Fault_RemoveClient(FaultClient* client) {
     }
 }
 
-void Fault_AddAddrConvClient(FaultAddrConvClient* client, void* callback, void* param) {
+/**
+ * Registers an address converter client. This enables the crash screen to look up virtual
+ * addresses of overlays relocated during runtime. Address conversion is carried out by
+ * `callback`, which either returns a virtual address or NULL if the address could not
+ * be converted.
+ *
+ * The callback is intended to be
+ * `uintptr_t (*callback)(uintptr_t addr, void* arg)`
```

Comment:
All of the current clients currently return `void*` so should probably update them to return `uintptr_t`s . Also I'm kind of thinking arg0 to the callback should remain an void*, since each client seems to do a cast to `uintptr_t` as one of the first things it does.

Reply:
Tbh, I think return `uintptr_t` is still better for these addrconv clients, since that is how Fault uses them always. 

Also looks like this comment still needs to be updated to match the typedef.

Reply:
Bump on the second comment.